drivers/unifiedPlatform/pds/backend/v1/api/restore.go

```diff
@@ -115,22 +115,22 @@ func (restore *PDS_API_V1) GetRestore(getRestoreRequest *automationModels.PDSRes
 	return &response, nil
 }
 
-// DeleteRestore will delete restore for a given deployment
-func (restore *PDS_API_V1) DeleteRestore(deleteRestoreRequest *automationModels.PDSRestoreRequest) error {
-	ctx, restoreClient, err := restore.getRestoreClient()
-	if err != nil {
-		return fmt.Errorf("Error while getting updated client with auth header: %v\n", err)
-	}
-
-	deleteRequest := restoreClient.RestoreServiceDeleteRestore(ctx, deleteRestoreRequest.Delete.Id)
-
-	_, res, err := deleteRequest.Execute()
-	if err != nil || res.StatusCode != status.StatusOK {
-		return fmt.Errorf("Error when calling `RestoreServiceGetRestore`: %v\n.Full HTTP response: %v", err, res)
-	}
-
-	return nil
-}
+//// DeleteRestore will delete restore for a given deployment
```

Comment:
these commented codes are not required ?

Reply:
No, Delete Restore is removed. But I have commented for now, if its needed later.